rebasehelper/cli.py

```diff
@@ -134,20 +139,28 @@ def run():
         except SystemExit as e:
             sys.exit(e.code)
         except BaseException:
-            if debug_log_file:
+            logger_traceback: CustomLogger = cast(CustomLogger, logging.getLogger('rebasehelper.traceback'))
+            logger_traceback.propagate = False
+            logger_traceback.setLevel(CustomLogger.TRACE)
+            if results_dir:
+                debug_log = os.path.join(results_dir, LOGS_DIR, DEBUG_LOG)
+                traceback_log = os.path.join(results_dir, LOGS_DIR, TRACEBACK_LOG)
                 logger.error('rebase-helper failed due to an unexpected error. Please report this problem'
                              '\nusing the following link: %s'
                              '\nand include the content of'
                              '\n\'%s\' and'
                              '\n\'%s\''
                              '\nin the report.'
                              '\nThank you!',
-                             NEW_ISSUE_LINK, debug_log_file, traceback_log)
+                             NEW_ISSUE_LINK, debug_log, traceback_log)
                 LoggerHelper.add_file_handler(logger_traceback, traceback_log)
                 logger_traceback.trace('', exc_info=1)
             else:
+                LoggerHelper.add_stream_handler(logger_traceback, CustomLogger.TRACE)
+                logger_traceback.trace('', exc_info=1)
```

Comment:
I think it looks better if the traceback follows the error message, not vice versa.

Reply:
I've thought about that, both look fine to me. But from the UX perspective - I usually look at the bottom of the terminal first and I expect the most important information to be there. Imho users won't try to examine the traceback. Also if the traceback is too long, it could bury the instructions for reporting the problem. There are definitely pros and cons to both approaches.

Reply:
Ok, but you should do something about the formatting. There is an empty line before the traceback, and no empty line between it and the actual error message.